src/app/app.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+angular.module('esn.calendar.event-consultation', [
+  'AngularJstz',
+  'ngRoute',
+  'ngSanitize',
+  'esn.avatar',
+  'esn.core',
+  'esn.calendar',
+  'esn.resource.libs',
+  'esn.calMoment',
+  'esn.constants',
+  'esn.core',
+  'esn.cache',
+  'esn.header',
+  'esn.http',
+  'esn.i18n',
+  'esn.ical',
+  'esn.lodash-wrapper',
+  'esn.notification',
+  'esn.object-type',
+  'esn.user',
+  'esn.url',  
+  'mgcrea.ngStrap.popover',
+  'ngSanitize',
+  'op.dynamicDirective',
+  'restangular',
+  'ngPromiseExtras',
+  'uuid4',
+  'openpaas-logo'
+]);
+
+require('esn-frontend-calendar/src/index.js');
```

Comment:
you should not require SPA index.js here but in the `./index.js`, 

Instead of, you should require `app.modules.js` here